mesa/visualization/ModularVisualization.md

```diff
@@ -67,25 +66,25 @@ Then, to launch a server with this grid as the only visualization element:
 ```
 ## Sub-Classing Modules
 
-In some cases, you may want or need to tweak the internals of an existing visualization module. The best way to do this is to create a subclass of it.
+If you want to tweak the internals of an existing visualization module, try subclassing it.
```

Comment:
Line 69: You might prefer something like a mix of your's and orignal's:

"In some cases, you may want to customize the internals of an existing visualization module. The best way to do this is to create a subclass of it."